agent/lib/sdk.js

```diff
@@ -107,8 +114,8 @@ const createSDK = (emitter) => {
     });
 
     const url = path.startsWith("/api")
-      ? [root, path].join("")
-      : [root, "api", version, "testdriver", path].join("/");
+      ? [config["TD_API_ROOT"], path].join("")
+      : [config["TD_API_ROOT"], "api", version, "testdriver", path].join("/");
```

Comment:
Same could be done here with:

```suggestion
    const url = new URL(
      path.startsWith("/api")
      ? path
      : `api/$version/testdriver/${path}`,
      config["TD_API_ROOT"]
    );  
```